server/status/feed.go

```diff
@@ -22,6 +22,79 @@ import (
 	"github.com/cockroachdb/cockroach/util"
 )
 
+// RegisterRangeEvent occurs in two scenarios. Firstly, while a store
+// broadcasts its list of ranges to initialize one or more new accumulators
+// (with Scan set to true), or secondly, when a new range is initialized on
+// the store (for example through replication), with Scan set to false. This
+// event includes the Range's RangeDescriptor and current StoreStats.
+type RegisterRangeEvent struct {
```

Comment:
I don't like the move of feed events from storage to server/status. This seems to point to a future in which server/status contains events from many feed publishers and becomes a sprawling mess like the monolithic proto package. These events (and the StoreStats proto) seem to belong closer to storage (perhaps a new storage/storestatus package if the relationship between storage and storage/engine makes things tricky).